### DISCUSSION TITLE
v4.9.0-rc.4

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5830] [Announcements]
Code v1.73.1

This is the fourth pre-release for v4.9.0. 

To install via the install script, pass the `--edge` flag. 

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.9.0-rc.4'>v4.9.0-rc.4</a>.</em>

**Comment by jsjoeio** [comment]
@benz0li this should have the correct file permissions for `product.json`. 

See details here: https://github.com/coder/code-server/discussions/5829#discussioncomment-4294743

And the fix here: https://github.com/coder/code-server/commit/0f66360646b842a620af2d5e7ce40da7f76b5931

**Reply by benz0li** [reply]
I can confirm it is fixed:

```bash
$ code-server --version          
4.9.0-rc.4 4db51ad34d4a3f7c7e1807b7f1c6b9d062139677 with Code 1.73.1
```

```bash
$ ls -al /opt/code-server/lib/vscode
total 168
drwxr-xr-x   7 root root  4096 Dec  2 19:09 .
drwxr-xr-x   3 root root  4096 Dec  2 19:08 ..
drwxr-xr-x   4 root root  4096 Dec  1 18:28 bin
drwxr-xr-x 106 root root  4096 Dec  3 09:09 extensions
drwxr-xr-x 107 root root  4096 Nov 30 23:30 node_modules
lrwxrwxrwx   1 root root    12 Dec  2 19:09 node_modules.asar -> node_modules
-rw-r--r--   1 root root 91042 Dec  1 20:30 npm-shrinkwrap.json
drwxr-xr-x   3 root root  4096 Dec  1 18:28 out
-rw-r--r--   1 root root  1222 Dec  1 20:30 package.json
-rwxr-xr-x   1 root root  3813 Dec  2 19:06 product.json
drwxr-xr-x   3 root root  4096 Dec  1 18:28 resources
-rw-r--r--   1 root root 38322 Dec  1 20:30 yarn.lock
```

**Reply by jsjoeio** [reply]
Hooray! Thank you!

**Comment by vikashDevZ** [comment]
Hi, I am looking for x86_64 supported file, can I get help?

**Reply by jsjoeio** [reply]
All the release assets for 4.9.0 can be found here: https://github.com/coder/code-server/releases/tag/v4.9.0
